lib/msal-common/src/account/AccountInfo.ts

```diff
@@ -105,12 +116,15 @@ export function buildTenantProfile(
             tenantId: tenantId,
             localAccountId: oid || sub || "",
             name: name,
+            username: preferred_username || upn || "",
+            loginHint: login_hint,
             isHomeTenant: tenantIdMatchesHomeTenant(tenantId, homeAccountId),
         };
     } else {
         return {
             tenantId,
             localAccountId,
+            username: "",
```

Comment:
I am curious, when do we expect to be building a TenantProfile without any idTokenClaims? Isn't an idToken a pre-requisite?